### PR TITLE
replay for 12_4_11

### PR DESCRIPTION
**Pull request by mdmorris**
# Replay Request

**Requestor**  
Meg Morris (for ORM)

**Describe the configuration**  
* Release: 12_4_11
* Run: 359762
* GTs:
   * expressGlobalTag: 124X_dataRun3_Express_v6
   * promptrecoGlobalTag:124X_dataRun3_Prompt_v5
   * alcap0GlobalTag: 124X_dataRun3_Prompt_v5
* Additional changes:

**Purpose of the test**  
Test the new CMSSW_12_4_11 release in view of the change in the T0 configuration

**T0 Operations cmsTalk thread**  
[Tier0 Operations cmsTalk Forum](https://cms-talk.web.cern.ch/c/offcomp/compops/tier0/210?)

### CODE REVIEW

**Comment by mmusich** [comment]
@mdmorris, I think the discussion in the ORM mattermost converged on adding 361239 as well.

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -131,9 +131,9 @@
 alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "124X_dataRun3_Express_v5"
-promptrecoGlobalTag = "124X_dataRun3_Prompt_v4"
-alcap0GlobalTag = "124X_dataRun3_Prompt_v4"
+expressGlobalTag = "124X_dataRun3_Express_v6"
```
Comment (tvami) [inline etc/ReplayOfflineConfiguration.py:134]:
```suggestion
expressGlobalTag = "124X_dataRun3_Express_v7"
```

File: etc/ReplayOfflineConfiguration.py [hunk 2]
```diff
@@ -131,9 +131,9 @@
...
+promptrecoGlobalTag = "124X_dataRun3_Prompt_v5"
```
Comment (tvami) [inline etc/ReplayOfflineConfiguration.py:135]:
```suggestion
promptrecoGlobalTag = "124X_dataRun3_Prompt_v6"
```

File: etc/ReplayOfflineConfiguration.py [hunk 3]
```diff
@@ -131,9 +131,9 @@
...
+alcap0GlobalTag = "124X_dataRun3_Prompt_v5"
```
Comment (tvami) [inline etc/ReplayOfflineConfiguration.py:136]:
```suggestion
alcap0GlobalTag = "124X_dataRun3_Prompt_v6"
```

**Review by tvami — COMMENTED** [review]
We have new GTs as well

**Comment by mdmorris** [comment]
switched back to run 359762
see mattermost conversation: https://mattermost.web.cern.ch/cms-exp/pl/p5ibis5qbtf65n3r4txs7rw7wh

**Comment by francescobrivio** [comment]
test syntax please

**Comment by germanfgv** [comment]
further tests on #4781

**Pull request closed without merging**